Form/Type/DocumentType.php

```diff
@@ -21,11 +21,7 @@
  */
 class DocumentType extends DoctrineType
 {
-    /**
-     * @param object $queryBuilder
-     * @param string $class
-     */
-    public function getLoader(ObjectManager $manager, $queryBuilder, $class): EntityLoaderInterface
+    public function getLoader(ObjectManager $manager, object $queryBuilder, string $class): EntityLoaderInterface
```

Comment:
[`DoctrineType::getLoader()` already has `object` and `string` in 5.4](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php#L234)